_posts/2021-04-20-custom-errors.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+published: true
+title:  Custom Errors in Solidity
+date:   2021-04-20
+author: Solidity Team
+category: Explainers
+---
+
+[Solidity v0.8.4](https://github.com/ethereum/solidity/releases/tag/v0.8.4) provides a convenient and gas-efficient way to explain to user why an operation failed. They can be defined inside and outside of contracts (including interfaces and libraries). 
+
+The syntax of Errors are similar to that of
```

Comment:
Maybe rather put this after the example? It could be confusing to mention events here, because only the syntax is similar, not the semantics.